internal/catalog/build_test.go

```diff
@@ -325,6 +325,26 @@ func TestUpdate(t *testing.T) {
 				},
 			},
 		},
+		{
+			`
+			CREATE FUNCTION foo(bar TEXT) RETURNS bool AS $$ SELECT true $$ LANGUAGE sql;
+			`,
+			pg.Catalog{
+				Schemas: map[string]pg.Schema{
+					"public": {
+						Enums:  map[string]pg.Enum{},
```

Comment:
For future tests, cmp.Diff is called with cmpopts.EquateEmpty(), so you don't need to create an empty map for the Enums and the Tables.